### DISCUSSION TITLE
Adjusting velocity

### DISCUSSION THREAD

**Discussion by cmakelabs** [larics/icuas24_competition#18] [Q&A] [answered]
Do all participants need to fly at a one specific and unified speed? If not, then what is the safest way to adjust the speed?

**Comment by fpetric** [comment, marked as answer]
You can fly at whatever speed you feel comfortable. I think the easiest way to change speed is to tinker with the controller params (startup/challenge/custom_config) in combination with your trajectory planner (or if you are using toppra, then check limits in the same folder mentioned earlier).
